src/platform/nxp/zephyr/ota/OTAImageProcessorImpl.h

```diff
@@ -40,11 +40,13 @@ class OTAImageProcessorImpl : public OTAImageProcessorInterface
     CHIP_ERROR ProcessBlock(ByteSpan & aBlock) override;
     bool IsFirstImageRun() override;
     CHIP_ERROR ConfirmCurrentImage() override;
+    void SetRebootDelay(uint16_t rebootDelay);
```

Comment:
Methods should also have unit of measurement in them like `SetRebootDelayMs` or use a typesafe unit type like Millisecond32 or similar.